docs/topics/genericgateway.md

```diff
@@ -47,18 +64,60 @@ When calling the Generic Gateway `simpleevent` endpoint, you must include a cust
 }
 ```
 
-`Ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, then your Build's `ref` will be set to `master`. If you do not wish to provide any payload, just include an empty JSON object (like `{}`).
+`ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, your Build's `ref` will be set to `master`. Of course, you can also provide any other values you need.
+If you do not wish to provide any payload, just send an empty JSON object (`{}`). 
```

Comment:
Is this working if I don't include data at all?
Last time I checked it failed - do we want it to work?

Reply:
this is on SimpleEvent, fixing it to allow empty payloads

Reply:
Sorry, I should have been a bit more specific - not adding an event object results in a failure - passing `{}` works. Is this actually the behaviour we want?

Reply:
yeah, I agree that it should allow for empty payloads, fixed it [here](https://github.com/dgkanatsios/brigade/blob/cloudevents/pkg/webhook/genericsimpleevent.go#L56) and the relevant test is [here](https://github.com/dgkanatsios/brigade/blob/cloudevents/pkg/webhook/genericsimpleevent_test.go#L195).